src/Sylius/Behat/Context/Api/Admin/ManagingExchangeRatesContext.php

```diff
@@ -0,0 +1,363 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Context\Api\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Sylius\Component\Currency\Model\CurrencyInterface;
+use Sylius\Component\Currency\Model\ExchangeRateInterface;
+use Sylius\Component\Currency\Repository\ExchangeRateRepositoryInterface;
+use Webmozart\Assert\Assert;
+
+final class ManagingExchangeRatesContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $client;
+
+    /** @var ExchangeRateRepositoryInterface */
+    private $exchangeRateRepository;
+
+    /** @var SharedStorageInterface */
+    private $sharedStorage;
+
+    public function __construct(
+        ApiClientInterface $client,
+        ExchangeRateRepositoryInterface $exchangeRateRepository,
+        SharedStorageInterface $sharedStorage
+    ) {
+        $this->client = $client;
+        $this->exchangeRateRepository = $exchangeRateRepository;
+        $this->sharedStorage = $sharedStorage;
+    }
+
+    /**
+     * @When I want to add a new exchange rate
+     */
+    public function iWantToAddNewExchangeRate(): void
+    {
+        $this->client->buildCreateRequest('exchange_rates');
+    }
+
+    /**
+     * @When /^I want to edit (this exchange rate)$/
+     */
+    public function iWantToEditThisExchangeRate(ExchangeRateInterface $exchangeRate): void
+    {
+        $this->sharedStorage->set('exchange_rate_id', $exchangeRate->getId());
```

Comment:
Why it's not the same step as the one below?

Reply:
Can you share the answer to Mateusz question?

Reply:
Because after this step I must repeat twice all flow from PUT request to check field immutablity in this two steps: https://github.com/Sylius/Sylius/blob/master/features/currency/managing_exchange_rates/editing_exchange_rate.feature#L23 
I talked about it with Mateusz

Reply:
But it doesn't change anything, I would be for merging these two steps because now, this step is confusing to me